### ISSUE TITLE
gripper_pad is not loaded to gazebo

### ISSUE THREAD

**Issue by knorth55** [start-jsk/jsk_apc#1893]
Require https://github.com/start-jsk/jsk_apc/pull/1892 and https://github.com/RethinkRobotics/baxter_simulator/pull/88

Gazebo simulator cannot load gripper pad.

![gazebo_apc_tuck](https://cloud.githubusercontent.com/assets/9300063/19262672/49e12118-8fd3-11e6-8236-54f154066561.png)
![gazebo_gripper_pad](https://cloud.githubusercontent.com/assets/9300063/19262673/4a162804-8fd3-11e6-9398-0542e3c5db3c.png)

**Comment by knorth55** [comment]
@pazeshun 
regenerate this condition
```
# baxter_simulator
roscd baxter_simulator
git remote add knorth55 https://github.com/knorth55/baxter_simulator.git
git fetch knorth55
git checkout knorth55/gripper-sim

# jsk_apc
roscd jsk_2016_01_baxter_apc
git remote add knorth55 https://github.com/knorth55/jsk_apc.git
git fetch knorth55
git checkout knorth55/fix-sim-apc2016

# roslaunch
roslaunch jsk_2016_01_baxter_apc baxter_pick_sim.launch
```

**Comment by pazeshun** [comment]
@knorth55 Thank you.
I'll try to fix this.

**Comment by knorth55** [comment]
#1940 i added gazebo tag, but it does not solve the problem.

**Comment by knorth55** [comment]
#1975 solve to load robot model, but not colored

**Comment by pazeshun** [comment]
I think http://gazebosim.org/tutorials/?tut=ros_urdf#NoteOnUnits will fix the problem

**Comment by knorth55** [comment]
Totally Fixed via #1975 
![baxter_gazebo_load](https://cloud.githubusercontent.com/assets/9300063/20961437/563935f2-bca8-11e6-9620-5d86d4b9ebea.png)
